### PR TITLE
feat: fix lora name bug, and generate on cloud not change error

### PR DESCRIPTION
**Pull request by o0OooO**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by NingLu** [comment]
We identified new issues on unchanged lines of code. Navigate to the Amazon CodeGuru Reviewer console to view the recommendations to fix them.

File: aws_extension/mme_utils.py [hunk 1]
```diff
@@ -105,24 +105,24 @@ def upload_model(model_type, model_name, model_s3_pos):
...
     logging.info(tar_name)
-    command = f"file --mime-type -b ./{tar_name}"
+    command = f'file --mime-type -b ./"{tar_name}"'
     file_type = subprocess.check_output(command, shell=True).decode('utf-8').strip()
     logging.info(f"file_type is {file_type}")
     if file_type == TAR_TYPE_FILE:
         logging.info("model type is tar")
-        os.system(f"tar xvf {tar_name}")
-        os.system(f"rm {tar_name}")
+        os.system(f"tar xvf '{tar_name}'")
```
Comment (NingLu) [inline aws_extension/mme_utils.py:117]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Starting a process with a shell, possible injection detected, security issue. https://bandit.readthedocs.io/en/latest/plugins/b605_start_process_with_a_shell.html

File: aws_extension/mme_utils.py [hunk 2]
```diff
@@ -105,24 +105,24 @@
...
+        os.system(f"rm '{tar_name}'")
```
Comment (NingLu) [inline aws_extension/mme_utils.py:118]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Starting a process with a shell, possible injection detected, security issue. https://bandit.readthedocs.io/en/latest/plugins/b605_start_process_with_a_shell.html

File: aws_extension/mme_utils.py [hunk 3]
```diff
@@ -105,24 +105,24 @@
...
         os.system("df -h")
     else:
-        os.system(f"rm {tar_name}")
+        os.system(f"rm '{tar_name}'")  # 使用引号括起文件名
         logging.info(f"model type is origin file type: {file_type}")
         prefix_name = model_s3_pos.split('.')[0]
-        os.system(f'./tools/s5cmd cp {prefix_name}* ./models/{model_type}/')
-        logging.info("download finished")
+        os.system(f'./tools/s5cmd cp "{prefix_name}"* ./models/{model_type}/')
```
Comment (NingLu) [inline aws_extension/mme_utils.py:124]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Starting a process with a shell, possible injection detected, security issue. https://bandit.readthedocs.io/en/latest/plugins/b605_start_process_with_a_shell.html

File: aws_extension/mme_utils.py [hunk 4]
```diff
@@ -105,24 +105,24 @@
 
 def download_and_update(model_type, model_s3_pos):
     #download from s3
-    logging.info(f'./tools/s5cmd cp {model_s3_pos} ./')
-    os.system(f'./tools/s5cmd cp {model_s3_pos} ./')
+    logging.info(f'./tools/s5cmd cp "{model_s3_pos}" ./')
+    os.system(f'./tools/s5cmd cp "{model_s3_pos}" ./')
```
Comment (NingLu) [inline aws_extension/mme_utils.py:109]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Starting a process with a shell, possible injection detected, security issue. https://bandit.readthedocs.io/en/latest/plugins/b605_start_process_with_a_shell.html

Comment (NingLu) [inline aws_extension/mme_utils.py:109]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This line of code makes an outdated API call to start and communicate with processes. We recommend that you use the `subprocess` module to start new processes, connect with their pipes, and get their return codes.

[Learn more](https://docs.python.org/3/library/subprocess.html)

Similar issue at line numbers 117, 118, 119, 121, and 124.

File: aws_extension/mme_utils.py [hunk 5]
```diff
@@ -105,24 +105,24 @@
...
     if file_type == TAR_TYPE_FILE:
         logging.info("model type is tar")
-        os.system(f"tar xvf {tar_name}")
-        os.system(f"rm {tar_name}")
+        os.system(f"tar xvf '{tar_name}'")
+        os.system(f"rm '{tar_name}'")
         os.system("df -h")
     else:
-        os.system(f"rm {tar_name}")
+        os.system(f"rm '{tar_name}'")  # 使用引号括起文件名
```
Comment (NingLu) [inline aws_extension/mme_utils.py:121]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Starting a process with a shell, possible injection detected, security issue. https://bandit.readthedocs.io/en/latest/plugins/b605_start_process_with_a_shell.html

**Review by guming3d — APPROVED** [review]

**Pull request merged**